### PR TITLE
replace line 174 sh:datatype xsd:string ;) by (new)sh:nodeKind sh:IRI

### PR DESCRIPTION
**Pull request by changqin26**
@JohannesLipp , a change request raised from Pascal. Please review. There the propertyShape for foaf:homepage has sh:datatype xsd:string ; but according to the property definition in the refactor_helper.ttl it should link an xsd:anyUri. Therefore, we would like to have there a sh:nodeKind sh:IRI ; instead of the string definition.

### CODE REVIEW

File: testing/participant/ParticipantShape.ttl [hunk 1]
```diff
@@ -171,7 +171,7 @@ shapes:PersonShape
 	sh:property [
 		a sh:PropertyShape ;
 		sh:path ids:homepage ;
-		sh:datatype xsd:string ;
+    sh:nodeKind sh:IRI ;
```
Comment (JohannesLipp) [inline testing/participant/ParticipantShape.ttl:174]:
```suggestion
    sh:datatype xsd:anyURI ;
```

Should be anyURI instead.

Comment (JohannesLipp) [reply testing/participant/ParticipantShape.ttl]: @changqin26

Comment (changqin26) [reply testing/participant/ParticipantShape.ttl]: Thanks for the hint, please review again.

Comment (changqin26) [reply testing/participant/ParticipantShape.ttl]: @JohannesLipp

**Review by JohannesLipp — APPROVED** [review]

**Pull request merged**